### ISSUE TITLE
Allow voice routing through hardware

### ISSUE THREAD

**Issue by Bluscream** [Splamy/TS3AudioBot#132]
It would be a neat addition if you could input/output from/to local audio devices

**Comment by Synergyst** [comment]
Please refer to issue #118

**Comment by Bluscream** [comment]
I'd say #118 is a dependant of this request then :)

**Comment by Synergyst** [comment]
Well, as of current you are able to route from a local recording device/stream with some modifications.
The way that I am doing it currently is not ideal since it is not native.

However there is currently no way to receive audio from the bot..

**Comment by Bluscream** [comment]
You mean through a rtmp stream?

**Comment by Synergyst** [comment]
That can work. However I changed where the bot looks for FFMPEG.
I made a BASH script to store the URL from the command and send it to FFMPEG or another utility such as SoX for recording from a local sound device or file.

It's essentially a simple hacked together wrapper script.
If you want to still have something like this before it's implemented natively then I can share it on a Gist after tidying it up.
